install_config/monitoring/overview.adoc

```diff
@@ -0,0 +1,35 @@
+= Overview
+:data-uri:
+:icons:
+:experimental:
+:prewrap!:
+
+OpenShift ships with a pre-configured and self-updating monitoring stack that is based on the link:https://prometheus.io/[Prometheus] open source project and its wider eco-system. It provides monitoring of cluster components and ships with a set of alerts to immediately notify the cluster administrator about any occurring problems and a set of link:https://grafana.com/[Grafana] dashboards.
+
+image::monitoring-diagram.png[]
+
+Highlighted in the diagram above, at the heart of the monitoring stack sits the OpenShift Cluster Monitoring Operator (CMO), which watches over the deployed monitoring components and resources, and ensures that they are always up to date.
+
+The Prometheus Operator (PO) creates, configures, and manages Prometheus and Alertmanager instances. It also automatically generates monitoring target configurations based on familiar Kubernetes label queries.
+
+In addition to Prometheus and Alertmanager, OpenShift Monitoring also includes link:https://github.com/prometheus/node_exporter[node-exporter] and link:https://github.com/kubernetes/kube-state-metrics[kube-state-metrics]. Node-exporter is an agent deployed on every node to collect metrics about it. The kube-state-metrics exporter agent converts Kubernetes objects to metrics consumable by Prometheus.
+
+The targets monitored as part of the cluster monitoring are:
+
+* Prometheus itself
+* Prometheus-Operator
+* Alertmanager cluster instances
+* Kubernetes apiserver
+* kubelets (the kubelet embeds cAdvisor for per container metrics)
+* kube-schedulers
```

Comment:
please change
kube-schedulers
to
kube-controllers

and add Cluster monitoring operator
see https://ibin.co/4HLU5Rnmqmg8.png

Reply:
Fixed